lib/explorer/data_frame.ex

```diff
@@ -1744,6 +1769,30 @@ defmodule Explorer.DataFrame do
         a integer [3, 3, 1]
         b integer [2, 3, 1]
       >
+
+  ## Grouped examples
+
+  When used in a grouped dataframe, `arrange_with/2` is going to sort each group individually
+  and then return the entire dataframe with the existing groups. If one of the arrange columns
```

Comment:
```suggestion
  and then return the entire dataframe with the existing groups. If one of the arranged columns
```

Or "one of the columns to arrange".